### ISSUE TITLE
What does works with universal mean?

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vsmarketplace#308]
This is shown on extensions:

![image](https://user-images.githubusercontent.com/2193314/149968246-9aafe106-4b40-4a21-ae1c-76e7638db8f6.png)

I'm assume this actually means desktop and am wondering why the ambiguous term "universal" is used. eg. I would expect "web" to be included in "universal".

**Comment by isidorn** [comment]
As @sandy081 and me already shared the term universal should not really be shown in the UI. It is just confusing.

**Comment by alefragnani** [comment]
I asked the same question a few days ago. The answer was https://github.com/microsoft/vsmarketplace/discussions/297#discussioncomment-1984117

**Comment by isidorn** [comment]
Yeah, we are highlighting here more of a design problem. We do understand what it means in theory :)
